atom/common/common_message_generator.h

```diff
@@ -1,10 +1,7 @@
-// Copyright (c) 2014 GitHub, Inc.
-// Use of this source code is governed by the MIT license that can be
-// found in the LICENSE file.
-
-// Multiply-included file, no traditional include guard.
-
-#include "atom/common/api/api_messages.h"
-#include "chrome/common/chrome_utility_printing_messages.h"
-#include "chrome/common/print_messages.h"
-#include "chrome/common/tts_messages.h"
```

Comment:
does this depend on the TTS PR being merged also?

Reply:
Not completely coupled with that, but there is a chromium equivalent of this message generator that already handles the TTS messages, and that caused a build conflict. Not sure how to resolve without removing these includes, since printing needs those messages from chromium. I guess I could patch that out of the chromium header, but again, then we would have to remove the patch when the TTS PR is merged. I'd love to hear some ideas.

Reply:
I don't mind if we just merge both PRs in quick succession. Looks like that's feasible.